doc/devel/add_new_projection.rst

```diff
@@ -16,13 +16,10 @@ that handle data in two or more dimensions at a time, are called
 "projections".
 
 From the user's perspective, the scale of a plot can be set with
-:meth:`~matplotlib.axes.Axes.set_xscale` and
-:meth:`~matplotlib.axes.Axes.set_yscale`.  Projections can be chosen
-using the ``projection`` keyword argument to the
-:func:`~matplotlib.pylab.plot` or :func:`~matplotlib.pylab.subplot`
-functions, e.g.::
+`.Axes.set_xscale` and `.Axes.set_yscale`.  Projections can be chosen using the
+*projection* keyword argument to the `.pyplot.subplot` function, e.g. ::
```

Comment:
Would it be good to mention that you can also pass `projection` to `pyplot.axes`?

Reply:
sure, done